### PR TITLE
Add a bio file for @githubteacher

### PR DESCRIPTION
**Pull request by githubteacher**
This Pull Request adds a bio file for @githubteacher as requested in #2.

Hey @githubstudent, I would :heart: if you had some :watch: to put some :eyes: on this and let me know your :thought_balloon:s.

### CODE REVIEW

**Comment by githubteacher** [comment]
This looks good for the most part.  There are a few issues we should resolve before merging.

File: docs./githubteacher.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+## GitHub Teacher
+
+- **Name:** GitHub Teacher
+- **Occupation:** Teacher
+- **Location:** San Francisco
```
Comment (githubteacher) [inline docs./githubteacher.md]: There are lots of San Franciscos in the :earth_africa:

File: docs./githubteacher.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+## GitHub Teacher
+
+- **Name:** GitHub Teacher
```
Comment (JohnShou) [inline docs./githubteacher.md:3]: How can you differentiate which comment is for which line

**Comment by githubteacher** [comment]
:+1: 

This looks good!  Let's :ship: it!

:shipit:

**Pull request merged**